docs/overview.md

```diff
@@ -0,0 +1,296 @@
+# Lily
+
+A Filecoin daemon designed to capture on-chain state from the Filecoin Network.
+
+## Overview
+
+At its core, Lily functions similarly to a Lotus node. It was named such as the Lily and Lotus flowers are often confused with each other. But, much like the flowers, these implementations are different.
+
+- How does Lily work?
+  - Lily is an application designed to capture on-chain-state from the Filecoin network. It runs as a daemon process wrapping a Lotus node, maintaining its own local blockstore, and synchronizing it with the Filecoin network. Lily uses its blockstore to efficiently index the Filecoin blockchain, and is capable of running different types of jobs: watch, walk, gap-find, and gap-fill. Each Job can run a variety of tasks that extract different parts of the chain. The data these tasks extract may be indexed in different storage back-ends: TimescaleDB and CSV files.
+- Who should use Lily?
+  - Users who wish to extract or inspect pieces of the Filecoin blockchain will find value in running a Lily node. The following examples are some of the uses cases Lily can fulfill:
+    - Indexing Blocks, Messages, and Receipts
+    - Deriving a view of on-chain Consensus
+    - Extracting miner sectors life-cycle events: Pre-Commit Add, Sector Add, Extend, Fault, Recovering, Recovered, Expiration, and Termination
+    - Collecting Internal message sends not appearing on chain. For example, Multisig actor sends, Cron Event Ticks, Block and Reward Messages.
+- But I don't want to run another daemon! Can I just get a copy the data?
+  - Fear not, an updated-daily archive of the information extracted by Lily is always available:
+    - https://lily-data.s3.us-east-2.amazonaws.com/ // TODO use an IPNS link
+
+## Build & Install
```

Comment:
Can we levarage lotus documentation so we don't have to maintain this (guessing it is the same?)?